src/Tribe/Templates.php

```diff
@@ -679,7 +679,25 @@ public static function event_date_to_pubDate( $time, $d, $gmt ) {
 			global $post;
 
 			if ( is_object( $post ) && $post->post_type == Tribe__Events__Main::POSTTYPE && is_feed() && $gmt ) {
-				$time = tribe_get_start_date( $post->ID, false, $d );
+
+				// Don't interfere if the timezone hasn't been set
+				$zone = get_option( 'timezone_string', false );
+				if ( false === $zone ) {
+					return $time;
+				}
+
+				// Get time and timezone
+				$time = new DateTime( tribe_get_start_date( $post->ID, false, $d ) );
+				$zone = new DateTimeZone( $zone );
+
+				// Apply timezone and calculate offset
+				$time->setTimezone( $zone );
+				$offset = $zone->getOffset( $time );
+				$offset *= - 2;
+				
+				// Apply correction as WordPress always outputs a pubDate set to 00:00 (UTC)
+				$time->modify( "$offset seconds" );
+				$time = $time->format( TribeDateUtils::DBDATETIMEFORMAT );
```

Comment:
I think this whole block could be simplified a bit.

We should be able to check if there is a timezone or not (like you are doing), then initialize a DateTimeZone object to be either `$zone` or `UTC`. Then set up the time object with a second parameter of `$zone`. Convert to UTC and set `$time` to the format we want.

Pseudo code:

```
if zone set
  $zone = DTZ( zone )
else
  $zone = DTZ( UTC )

$time = DT( $time, $zone )
$time->setTimezone( DTZ( UTC ) )
$time = $time->format( $d )
// and mysql2date would be ditchable
```